kie-server-parent/kie-server-remote/kie-server-rest/kie-server-rest-jbpm/src/main/java/org/kie/server/remote/rest/jbpm/FormResource.java

```diff
@@ -0,0 +1,122 @@
+package org.kie.server.remote.rest.jbpm;
+
+import static org.kie.server.api.rest.RestURI.CONTAINER_ID;
+import static org.kie.server.api.rest.RestURI.FORM_URI;
+import static org.kie.server.api.rest.RestURI.PROCESS_FORM_URI;
+import static org.kie.server.api.rest.RestURI.PROCESS_ID;
+import static org.kie.server.api.rest.RestURI.TASK_FORM_URI;
+import static org.kie.server.api.rest.RestURI.TASK_INSTANCE_ID;
+import static org.kie.server.remote.rest.common.util.RestUtils.createResponse;
+import static org.kie.server.remote.rest.common.util.RestUtils.getVariant;
+import static org.kie.server.remote.rest.common.util.RestUtils.internalServerError;
+import static org.kie.server.remote.rest.jbpm.resources.Messages.UNEXPECTED_ERROR;
+
+import java.text.MessageFormat;
+
+import javax.ws.rs.DefaultValue;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.HttpHeaders;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.Variant;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.json.XML;
+import org.kie.server.services.jbpm.FormServiceBase;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Path("server/" + FORM_URI)
+public class FormResource {
+
+    public static final Logger logger = LoggerFactory.getLogger(FormResource.class);
+    public static int PRETTY_PRINT_INDENT_FACTOR = 4;
+
+    private FormServiceBase formServiceBase;
+
+    public FormResource() {
+
+    }
+
+    public FormResource(FormServiceBase formServiceBase) {
+        this.formServiceBase = formServiceBase;
+    }
+
+    @GET
+    @Path(PROCESS_FORM_URI)
+    @Produces({ MediaType.APPLICATION_JSON })
+    public Response getProcessForm(@javax.ws.rs.core.Context HttpHeaders headers,
+            @PathParam(CONTAINER_ID) String containerId, @PathParam(PROCESS_ID) String processId,
+            @QueryParam("lang") @DefaultValue("en") String language) {
```

Comment:
If look at any label, title, error message in the form file.... it looks like this <property name="label" value="quot;SSClassquot;,quot;quot;quot;SSStylequot;,quot;quot;quot;caquot;,quot;quot;quot;dequot;,quot;quot;quot;enquot;,quot;Agequot;quot;esquot;,quot;quot;quot;frquot;,quot;quot;quot;jaquot;,quot;quot;quot;ptquot;,quot;quot;quot;zhquot;,quot;quot;"/>

The purpose of this parameter is to filter these properties, so they would contain only a text representing the value. In this example after filtering the property would look like:

<property name="label" value="Age"/>

which is much easier to work with.